linchpin/provision/roles/azure/tasks/provisionVM.yml

```diff
@@ -1,6 +1,8 @@
 ---
 - name: "Provisioning Azure VM when not async"
   azure_rm_virtualmachine:
+    ad_user: "{{ auth_var['user'] | default(omit) }}"
+    password: "{{ auth_var['password'] | default(omit) }}"
     client_id: "{{ auth_var['client_id'] | default(omit) }}"
```

Comment:
Didn't delete old creds like (client_id, subscription_id), ansible will only detect one way of authentication

Reply:
I left it on purpose for "IAM" method of authentication, checkout the doc.